README.md

```diff
@@ -1333,6 +1333,7 @@ _Related: [Wikis](#wikis)_
 - [draw.io](https://draw.io) - Diagram software for making flowcharts, process diagrams, org charts, UML, ER and network diagrams. ([Source Code](https://github.com/jgraph/drawio)) `Apache-2.0` `Javascript`
 - [HedgeDoc](https://demo.hedgedoc.org/) - Realtime collaborative markdown notes on all platforms, formerly known as CodiMD and HackMD CE. ([Source Code](https://github.com/hedgedoc/hedgedoc)) `AGPL-3.0` `Docker/Nodejs`
 - [Joplin](https://joplinapp.org/) - Joplin is a note taking application with Markdown editor and encryption support for mobile and desktop platforms. Runs client-side and syncs through self hosted Nextcloud or similar. Consider it like open source alternative to Evernote. ([Source Code](https://github.com/laurent22/joplin)) `MIT` `Nodejs`
+- [Kiwix](https://www.kiwix.org/en/) - Wherever you go, you can browse Wikipedia, read books from the Gutenberg Library, or watch TED talks and much more – even if you don’t have an Internet connection. ([Source Code](https://github.com/kiwix/kiwix-tools)) `GPL-3.0` `C++`
```

Comment:
I assume you want to add the `kiwix-serve: HTTP daemon serving ZIM files` program, not the kiwix desktop application (which does not qualify for this list).

In that case it's better to change the main lin k to https://www.kiwix.org/en/downloads/kiwix-serve/, and the description to `HTTP daemon for serving wikis from ZIM files`

(https://en.wikipedia.org/wiki/ZIM_(file_format))

Reply:
Am I suppose to use the last Wikipedia link anywhere?

Reply:
No, it's fine as-is, thank you.